egg-herbie/main.rkt

```diff
@@ -22,7 +22,40 @@
                  [else "libegg_math"])
                 (bytes->string/utf-8 (system-type 'so-suffix)))))
 
-(define-ffi-definer define-eggmath (ffi-lib libeggmath-path))
+; Checks if Racket is being run in emulation via rosetta
+(define (running-on-rosetta?)
+  (if (and (equal? (system-type 'os) 'macosx)
+           (equal? (system-type 'arch) 'x86_64))
+      (equal? (with-output-to-string (lambda () ; returns true if running in emulation
+                                       (system "sysctl -n sysctl.proc_translated"))) "1\n")
+      #f))
+
+; Note, this message should not be reached.
+(define fallback-message
+  (string-join
+   `("Error: unable to load the 'egg-math' library"
+     "Please file a bug at \"https://github.com/herbie-fp/herbie/issues\"")

```

Comment:
Also, I would drop the quotes around the URL; you don't have them in the other message.